recipes/dask-cuda/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "dask-cuda" %}
+{% set version = "0.18.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dask-cuda-{{ version }}.tar.gz
+  sha256: f5127667bbb6730e23e22016fb1a64f308d8e8455fa61993990e5d237df75396
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - dask-cuda-worker = dask_cuda.cli.dask_cuda_worker:go
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
```

Comment:
I know very little on dask-cuda, but shouldn't `cudatoolkit` be one of the run dependencies?

Reply:
No it doesn't need `cudatoolkit`. It uses `numba`, which doesn't need this